swagger.yaml

```diff
@@ -1,75 +1,208 @@
-    in: query
-    type: boolean
-    required: false
-    description: If true, JSON response will be pretty-printed
+        - name: "course"
+          in: "query"
+          description: "Name of course in which to find textbooks"
+          type: "string"
+          required: true
+
+        - name: "section"
+          in: "query"
+          description: "Name of section in which to find textbooks"
+          type: "string"
+          required: false
+
+        - name: "isRequired"
+          in: "query"
+          description: "If true, only retrieves required textbooks"
+          type: "string"
+          required: false
+          default: false
+
+      responses:
+        200:
+          description: "Textbooks successfully retrieved"
+          schema:
+            $ref: "#/definitions/TextbookResultArray"
+        400:
+          description: "Bad request"
+          schema:
+            $ref: "#/definitions/Error"
+        500:
+          description: "Internal server error"
+          schema:
+            $ref: "#/definitions/Error"
 
 definitions:
-  Info:
+
+  Textbook:
+    type: "object"
     properties:
-      name:
-        type: string
-        description: Name of the API.
-      time:
-        type: string
-        description: Build time in ISO 8601 format.
-      unixTime:
-        type: integer
-        description: Build time as milliseconds since the unix epoch.
-      commit:
-        type: string
-        description: Git commit hash of the revision which was built.
-      documentation:
-        type: string
-        description: Filename of the swagger specification for the API.
+      isbn:
+        type: "string"
+        description: "ISBN of textbook"
+        example: "9780998819112"
+      required:
+        type: "string"
+        description: "Required status of textbook"
+        example: "Required"
+      coverImageUrl:
+        type: "string"
+        description: "URL of cover image"
+        example: "coverimages.verbacompete.com/no_image.jpg"
+      title:
+        type: "string"
+        description: "Title of textbook"
+        example: "Probability & Stat For Engr & Sci"
+      author:
+        type: "string"
+        description: "Author of textbook"
+        example: "Devore"
+      edition:
+        type: "string"
+        description: "Textbook edition"
+        example: "9"
+      copyright_year:
+        type: "string"
+        description: "Textbook copyright year"
+        example: "2016"
+      priceNew:
+        type: "string"
+        description: "Price to buy new textbook at Beaver Store"
+        example: "$49.00"
+      priceUsed:
+        type: "string"
+        description: "Price to buy used textbook at Beaver Store"
+        example: "$36.75"
+
+  TextbookResource:
+    type: "object"
+    properties:
+        id:
+          type: "string"
+          description: "ID of textbook given by section id and textbook ISBN"
+          example: "2018-Fall__ST__314__002__9780998819112"
```

Comment:
I know this just an example, but is there any way we can make this more "pretty"? For example, use only dashes instead of underscores.